src/api.v2/controllers/gem2sController.js

```diff
@@ -9,8 +9,8 @@ const ExperimentParent = require('../model/ExperimentParent');
 
 const logger = getLogger('[Gem2sController] - ');
 
-const runGem2s = async (req, res) => {
-  const { experimentId } = req.params;
+const runGem2s = async (stateMachineParams, authorization) => {
```

Comment:
I think just calling these `params` is better (or passing each of the params individually). They can come from the ui or from the retry so they are not necessarily state machine.

Also the state machine params are generated by using these, but these are not the actual `stateMachineParams`, right? 

This comment applies to the counterparts for subset and seurat as well

Reply:
Renamed